tests/30-main.spec.js

```diff
@@ -254,4 +255,450 @@ describe('main', () => {
       credential, documentLoader, verifyRevocationListCredential: false});
     result.verified.should.equal(false);
   });
+
+  it('should fail to verify status on missing "credential" param', async () => {
+    let err;
+    let result;
+    try {
+      result = await checkStatus({
+        documentLoader, verifyRevocationListCredential: false});
+      result.verified.should.equal(false);
+    } catch(e) {
+      err = e;
+    }
+    should.not.exist(err);
+    should.exist(result);
+    result.should.be.an('object');
+    result.should.have.property('verified');
+    result.verified.should.be.a('boolean');
+    result.verified.should.be.false;
+    result.should.have.property('error');
+    result.error.should.be.instanceof(TypeError);
+    result.error.message.should.contain('"credential" must be an object');
+  });
+
+  it('should fail to verify if credential is not an object for ' +
+   'statusTypeMatches"', async () => {
+    let err;
+    let result;
+    try {
+      result = statusTypeMatches({credential: ''});
+    } catch(e) {
+      err = e;
+    }
+    should.exist(err);
+    should.not.exist(result);
+    err.should.be.instanceof(TypeError);
+    err.message.should.contain('"credential" must be an object');
+  });
+
+  it('should fail to verify if credential is not an object for ' +
+    '"assertRevocationList2020Context"', async () => {
+    let err;
+    let result;
+    try {
+      result = assertRevocationList2020Context('{credential}');
```

Comment:
There are still issues like this one that were flagged in the last PR that haven't been addressed.

Reply:
@LoveishSarolia Please fix outstanding issues.

Reply:
I believe I fixed them a while ago and pushed them to the #add-tests-pr branch. If there are any other issues, I am not seeing them. Am I correct in thinking these are the only issues that needed to be addressed?